ui/arduino/helpers.py

```diff
@@ -30,3 +40,5 @@ def delete_folder(path):
         if file['type'] == 'folder':
             os.rmdir(file['path'])
     os.rmdir(path)
+
+os.chdir(get_root())
```

Comment:
It should not be necessary to change to the root. The system should be mounted by default there. Unless someone is using some code that changes that. Then again, the board does a soft reset when connecting. Can you elaborate?

Reply:
> Unless someone is using some code that changes that

exactly this